### ISSUE TITLE
VPC-Serverless-Connector-Beta changes forces replacement

### ISSUE THREAD

**Issue by sclausson** [terraform-google-modules/terraform-google-network#547]
### TL;DR

According to the docs you should be able update a running VPC Connector (since Jan 23)
https://cloud.google.com/run/docs/configuring/vpc-connectors#update-connector. 

In my case I am just trying to change Max Instances from 3 to 5, which forces replacement of VPC Connector

### Additional information

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
-/+ destroy and then create replacement

Terraform will perform the following actions:

  # google_vpc_access_connector.connector_beta["vpc-conn"] must be replaced
-/+ resource "google_vpc_access_connector" "connector_beta" {
      ~ connected_projects = [
          - "omitted",
        ] -> (known after apply)
      ~ id                 = "projects/<...>/locations/europe-west1/connectors/<...>" -> (known after apply)
      ~ max_instances      = 3 -> 5 **# forces replacement**
        name               = "omitted"
      ~ network            = "xxx-network" -> (known after apply)
      ~ self_link          = "projects/<...>/locations/europe-west1/connectors/<...>" -> (known after apply)
      ~ state              = "READY" -> (known after apply)
        # (6 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 1 to add, 0 to change, 1 to destroy.

**Comment by imrannayer** [comment]
@sclausson It seems like a provider issue. Is it possible if you can create an issue for provider [here](https://github.com/hashicorp/terraform-provider-google/issues)? Mention this Issue so we get updates here when they update the issue?
